processor/agentmetricsprocessor/agentmetricsprocessor.go

```diff
@@ -78,6 +80,11 @@ func (mtp *agentMetricsProcessor) ProcessMetrics(ctx context.Context, metrics pd
 		errors = append(errors, err)
 	}
 
+	// Add blank labels last so it can be used on newly-generated metrics.
```

Comment:
Can you please expand on this comment? What is the connection between newly-generated metrics and adding the blank label last?

Reply:
I'm not sure what other words I can use to explain this; agentmetricsprocessor creates new metrics, and we need to be able to add a blank label to one of those metrics.

Reply:
How about: `// Add blank labels last so they also apply to metrics added by agentmetricsprocessor.`?

Reply:
Done.